guides/source/configuring.md

```diff
@@ -668,6 +668,7 @@ Defaults to `'signed cookie'`.
 
 * `config.action_view.default_enforce_utf8` determines whether forms are generated with a hidden tag that forces older versions of Internet Explorer to submit forms encoded in UTF-8. This defaults to `false`.
 
+* `config.action_view.annotate_template_file_names` determines whether to annotate HTML output generated by Rails with file names of the templates. Supports HTML and ERB templates. This defaults to `false`.
```

Comment:
```suggestion
* `config.action_view.annotate_template_file_names` determines whether to annotate HTML with template file names. This defaults to `false`.
```

Suggested copy change ☝️ . Thanks for catching this!

For what it’s worth, other renderers will likely support this functionality soon.

Reply:
@joelhawksley Is ERB not supported? I thought it is already supported based on the code.